README.md

```diff
@@ -21,31 +25,14 @@ def deps do
 end
 ```
 
-## Basic Usage
+<!-- MDOC !-->
```

Comment:
This looks hacky, not sure if I like this.

Reply:
> This looks hacky, not sure if I like this.

Removed.